File: project_6/mario.py
from cs50 import get_int

def main():
    h = get_positive_integer()

    for i in range(h):
        for j in range(i, h - 1):
            print(" ", end="")
        for k in range(i + 1):
            print("#", end="")
        print()

def get_positive_integer():
    while True:
        height = get_int("Height: ")
        if height <= 8 and height >=1:
            break
    return height

main()

File: project_1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("What height is your pyramid? ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {

        for (int j = i; j < height - 1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }


}